src/components/TaskList/TaskList.jsx

```diff
@@ -1,20 +1,31 @@
 const React = require('react')
+const { connect } = require('react-redux')
+
+const Task = require('../Task')
 
 class TaskList extends React.Component {
+  onSnoozeClick() {
+    this.props.dispatch({ type: 'TASKS_SNOOZE' })
+  }
+
   render() {
     return (
       <div>
         <nav className="controls-container">
-          <button type="button" className="control">snooze</button>
+          <button id="snooze" type="button" onClick={() => this.onSnoozeClick()} className="control">snooze</button>
           <button type="button" className="control">ignore</button>
           <button type="button" className="control">archive</button>
         </nav>
         <ol className="issues-list">
-          <li>LOOK AT THIS ISSUE</li>
+          {this.props.tasks.map(task => {
+            return (<Task key={task.id} task={task} />)
+          })}
         </ol>
       </div>
     )
   }
 }
 
-module.exports = TaskList
+const mapStateToProps = tasks => ({ tasks })
+
+module.exports = connect(mapStateToProps)(TaskList)
```

Comment:
The best way is to read http://redux.js.org/index.html :wink: but I'll try a tl;dr.

With Redux, an app has one object that holds every bit of state. You can pass this state from component to component via props, but that gets a little cumbersome. So in redux people use `connect`. It gives a component access to the global state by adding it to the components props. If you don't want to give the component access to the entire state, you can reduce what it has access to using `mapStateToProps`.

After explaining that I realize I don't need to use mapStateToProps here, so I'm going to remove it.